Subdivision_method_3/include/CGAL/Subdivision_method_3/subdivision_methods_3.h

```diff
@@ -120,6 +118,13 @@ void CatmullClark_subdivision(PolygonMesh& pmesh, int step) {
  *     \cgalParamType{unsigned int}
  *     \cgalParamDefault{`1`}
  *   \cgalParamNEnd
+ *   \cgalParamNBegin{do_not_modify_geometry}
```

Comment:
I would prefer that the default was `true` and that the named parameter is just `modify_geometry()`.